### ISSUE TITLE
nbTree

### ISSUE THREAD

**Issue by neroist** [pietroppeter/nimib#233]
Hello, its been a bit, however I was looking at [this](https://stackoverflow.com/a/31087670/14586140) stackoverflow answer and I thought it would be a cool block for nimib to have (perhaps instead something for nimibex?)

**Comment by pietroppeter** [comment]
yep, while in speaking hours with @HugoGranstrom we still agree it could be a nice thing for nimibex. only problem is we have not made progress on nimibex so it's harder to contribute there. let us know if there is something we can help with and anyway thanks for the suggestion!
